From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Fri, 11 Mar 2022 12:45:28 -0500
Subject: [PATCH] make: handle some gcc -Wanalyzer flags better

This makes it so we won't use the -Wanalyzer / -fanalyzer flags by
default, because they're still pretty overzealous.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 Make.defaults | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

NOTE: This change is already present in pesign 116. The patch body is a no-op so the packaging series continues to apply cleanly after the version bump.

diff --git a/TODO b/TODO
--- a/TODO
+++ b/TODO
@@ -1 +1 @@
-Stuff that needs to get done:
+Stuff that needs to get done:
